### PR TITLE
New account: hide UI tips

### PR DESCRIPTION
**Pull request by breville**
When a new account is created, set the properties for the UI tips shown for teacher home & courses pages to be "dismissed" so that they aren't shown.  That's because we don't need to show these tips to new users, only returning ones.

### CODE REVIEW

**Comment by breville** [comment]
cc @tanyaparker

**Comment by Erin007** [comment]
hamburger UI tests go looking for the x to close the dialog for uitips when a new teacher is created; just delete that check and you should be good to go.

File: dashboard/app/models/user.rb [hunk 1]
```diff
@@ -333,6 +333,8 @@ def district_teachers(cohort = nil)
 
   before_create :generate_secret_words
 
+  before_create :clear_ui_tips
```
Comment (aoby) [inline dashboard/app/models/user.rb:336]: nit: This naming seems odd to me. It doesn't make sense to "clear" something on a new user.

How about something like `suppress_ui_tips_for_new_users`, which is more self-explanatory?

Comment (breville) [reply dashboard/app/models/user.rb:336]: Addressed in https://github.com/code-dot-org/code-dot-org/pull/15968

**Pull request merged**